.github/workflows/run_tests.yml

```diff
@@ -35,12 +35,21 @@ jobs:
 
       - name: Install dependencies
         run: |
+          if [ "$RUNNER_OS" == "Windows" ]; then
+              pip install pywin32==306
+          fi
           pip install ecdsa==0.18.0 sympy==1.11.1
+        shell: bash
```

Comment:
windows does not have bash, so. I'm not sure how this works :D nevertheless, let's rewrite it to windows specific step, similarly to this: https://github.com/software-mansion-labs/crypto-cpp-py/blob/main/.github/workflows/package.yml#L23